test/example_migrations/20220726000151_create_index_concurrently_valid.exs

```diff
@@ -1,18 +0,0 @@
-defmodule ExcellentMigrations.CreateIndexConcurrently do
-  @disable_ddl_transaction true
-  @disable_migration_lock true
-
-  def up do
-    create(index(:dumplings, [:dough], concurrently: true))
-    create_if_not_exists(index(:dumplings, [:dough], concurrently: true))
-    create(unique_index(:dumplings, [:dough], concurrently: true))
-    create_if_not_exists(unique_index(:dumplings, [:dough], concurrently: true))
-  end
-
-  def down do
-    drop(index(:dumplings, [:dough], concurrently: true))
-    drop_if_exists(index(:dumplings, [:dough], concurrently: true))
-    drop(unique_index(:dumplings, [:dough], concurrently: true))
-    drop_if_exists(unique_index(:dumplings, [:dough], concurrently: true))
-  end
```

Comment:
One thing that comes to my mind now when looking at this change is that dropping index could be considered unsafe only if done in `up` or `change`, while in `down` it's actually fine.